main.js

```diff
@@ -0,0 +1,54 @@
+const electron = require("electron");
+// Module to control application life.
+const {app} = electron;
```

Comment:
Why did the curly braces come back? I'm assuming they're needed, it'd be nice to know why and possibly add a comment